src/libopensc/pkcs15-idprime.c

```diff
@@ -98,7 +98,8 @@ static int sc_pkcs15emu_idprime_init(sc_pkcs15_card_t *p15card)
 	pin_info.attrs.pin.max_length    = 16;
 	pin_info.tries_left    = -1;
 
-	if (card->type == SC_CARD_TYPE_IDPRIME_V3) {
+	if (card->type == SC_CARD_TYPE_IDPRIME_V3 ||
+		card->type == SC_CARD_TYPE_IDPRIME_V4) {
```

Comment:
just one more tab and you're in line with the rest of *our* code base

Reply:
Sorry about that! Should be fine now!